.ameba.yml

```diff
@@ -88,9 +88,20 @@ Lint/UnusedArgument:
 Lint/UnusedBlockArgument:
   Enabled: false
 
-# FIXME: Investigate useless assigns.
 Lint/UselessAssign:
-  Enabled: false
+  # BUG: https://github.com/crystal-ameba/ameba/issues/447
+  # This setting is to avoid false positives from the common use of type
+  # declarations in macro arguments.
+  ExcludeTypeDeclarations: true
+  Excluded:
+    - spec/debug/**/*
+    - samples/mt_gc_test.cr
+    # BUG: https://github.com/crystal-ameba/ameba/issues/624
```

Comment:
That's already fixed in `master`.

Reply:
This config already assumes unreleased version, just an earlier commit, might be bumped as well - in this or another PR

Reply:
But I don't want to chase unreleased versions and figure out which config is necessary for which one. That's just time-consuming without much benefit. Let's wait for the next ameba release and update the config then.